.github/workflows/php.yml

```diff
@@ -107,6 +107,7 @@ jobs:
         composer require wikimedia/composer-merge-plugin
         composer config --json extra.merge-plugin.require '["modules/contrib/apigee_m10n/composer.json"]'
         composer config platform.php ${{ matrix.php-version }}
+        composer config --json extra.patches."drupal/core" '{ "Solves Deprecated function: preg_quote()": "https://www.drupal.org/files/issues/2023-11-10/3327662-1_1.patch"}'
```

Comment:
Please add the same title as the core issue https://www.drupal.org/project/drupal/issues/3327662